x-pack/plugins/aiops/public/components/log_categorization/log_categorization_for_embeddable/log_categorization_for_embeddable.tsx

```diff
@@ -354,12 +354,17 @@ export const LogCategorizationEmbeddable: FC<LogCategorizationEmbeddableProps> =
     [lastReloadRequestTime]
   );
 
-  const actions = [...getActions(false), ...getActions(true)];
+  const isCasesEmbeddable = embeddingOrigin === AIOPS_EMBEDDABLE_ORIGIN.CASES;
+
+  // Disable filtering for cases embeddable, as it's not possible to delete filters
+  const actions = [...(isCasesEmbeddable ? [] : getActions(false)), ...getActions(true)];
 
   return (
     <>
       <FieldValidationCallout validationResults={fieldValidationResult} />
-      {(loading ?? true) === true ? <LoadingCategorization onCancel={cancelRequest} /> : null}
+      {(loading ?? true) === true ? (
+        <LoadingCategorization onCancel={cancelRequest} disableCancelLoading={isCasesEmbeddable} />
```

Comment:
Disabled `cancel loading` in cases because it is not possible to refresh the embeddable. Therefore, after canceling, the embeddable would get stuck without results.